include/irr/asset/ICPUVirtualTexture.h

```diff
@@ -55,6 +55,80 @@ class ICPUVirtualTexture final : public IVirtualTexture<ICPUImageView, ICPUSampl
         }
     };
 
+    static auto createPoTPaddedSquareImageWithMipLevels(const asset::ICPUImage* _img, asset::ISampler::E_TEXTURE_CLAMP _wrapu, asset::ISampler::E_TEXTURE_CLAMP _wrapv)
```

Comment:
because mip-map filter may have a bigger domain than [-0.5,0.5] it is also important to know the border color